e2e/list.test.ts

```diff
@@ -0,0 +1,51 @@
+import { ensureProject, forEachCli, runCLI } from './utils';
+
+const testTimeout = 120000;
+
+forEachCli('nx', () => {
+  describe('list', () => {
+    beforeEach(() => {
+      ensureProject();
+    });
+
+    it(
+      `should list available collections`,
+      async () => {
+        const listOutput = runCLI('list');
+
+        expect(listOutput).toContain('Available collections');
+        // just check for some, not all
+        expect(listOutput).toContain('@nrwl/angular');
+        expect(listOutput).toContain('@schematics/angular');
+        expect(listOutput).toContain('@ngrx/store');
+      },
+      testTimeout
+    );
+
+    it(
+      `should list available schematics in a collection`,
+      async () => {
+        const listOutput = runCLI('list @nrwl/angular');
+
+        expect(listOutput).toContain('Available schematics in @nrwl/angular');
+        // just check for some, not all
+        expect(listOutput).toContain('init');
+        expect(listOutput).toContain('application');
+        expect(listOutput).toContain('library');
+      },
+      testTimeout
+    );
+  });
+});
+
+forEachCli('angular', () => {
+  describe('list', () => {
+    it(
+      `shouldn't do anything at all`,
```

Comment:
Since the implementaiton isn't in tao but in workspace, the command should work with both the Tao and the Angular CLIs, right?

Reply:
How? It's like `dep-graph` or similar in that it only exists on `nx`...

I thought `tao` only contained commands that also exist in the Angular CLI?

Reply:
Changed as per VC last night. I did consider renaming `forEachCli` to something like `forEachWorkspace` but thought that might be a bit much as an incidental change...